packages/react-native/ReactAndroid/src/test/java/com/facebook/react/internal/interop/InteropEventEmitterTest.kt

```diff
@@ -40,9 +39,9 @@ class InteropEventEmitterTest {
 
     eventEmitter.receiveEvent(42, "onTest", null)
 
-    assertEquals(1, eventDispatcher.getRecordedDispatchedEvents().size)
-    assertEquals("onTest", eventDispatcher.getRecordedDispatchedEvents().get(0).getEventName())
-    assertEquals(InteropEvent::class, eventDispatcher.getRecordedDispatchedEvents().get(0)::class)
+    assertThat(eventDispatcher.getRecordedDispatchedEvents().size).isEqualTo(1)
+    assertThat(eventDispatcher.getRecordedDispatchedEvents().get(0).getEventName()).isEqualTo("onTest")
+    assertThat(eventDispatcher.getRecordedDispatchedEvents().get(0)::class).isEqualTo(InteropEvent::class)
```

Comment:
Same instance of check

Reply:
Hey @cortinico.

Tests fail when using .instanceOf().

`assertThat(eventDispatcher.getRecordedDispatchedEvents().get(0)::class.java).isInstanceOf(InteropEvent::class.java)`

Any idea?

```
Expecting actual:
  com.facebook.react.internal.interop.InteropEvent
to be an instance of:
  com.facebook.react.internal.interop.InteropEvent
but was instance of:
  java.lang.Class
java.lang.AssertionError: 
Expecting actual:
  com.facebook.react.internal.interop.InteropEvent
to be an instance of:
  com.facebook.react.internal.interop.InteropEvent
but was instance of:
  java.lang.Class
	at com.facebook.react.internal.interop.InteropEventEmitterTest.receiveEvent_dispatchesCorrectly(InteropEventEmitterTest.kt:44)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.robolectric.RobolectricTestRunner$HelperTestRunner$1.evaluate(RobolectricTestRunner.java:580)
	at org.robolectric.internal.SandboxTestRunner$2.lambda$evaluate$2(SandboxTestRunner.java:287)
	at org.robolectric.internal.bytecode.Sandbox.lambda$runOnMainThread$0(Sandbox.java:99)
```

Reply:
The correct assertion shoudl be:

```
assertThat(eventDispatcher.getRecordedDispatchedEvents().get(0)).isInstanceOf(InteropEvent::class.java)
```